atomicapp/providers/docker.py

```diff
@@ -51,15 +60,41 @@ def init(self):
             raise ProviderFailedException(msg)
 
     def deploy(self):
+        logger.info("Deploying to provider: Docker")
+
         for artifact in self.artifacts:
             artifact_path = os.path.join(self.path, artifact)
             label_run = None
             with open(artifact_path, "r") as fp:
                 label_run = fp.read().strip()
+            labels = label_run.split()
```

Comment:
I am sorry that I haven't realized it earlier, but could you rename `labels` to something more matching what it is? Like `run_args` f.e.?